src/ape/managers/project.py

```diff
@@ -467,19 +467,16 @@ def __getattr__(self, attr_name: str) -> ContractContainer:
             :class:`~ape.contracts.ContractContainer`
         """
 
-        contracts = self.load_contracts()
-        if attr_name in contracts:
-            contract_type = contracts[attr_name]
-        elif attr_name in self.dependencies:
-            contract_type = self.dependencies[attr_name]  # type: ignore
-        else:
-            # Fixes anomaly when accessing non-ContractType attributes.
-            # Returns normal attribute if exists. Raises 'AttributeError' otherwise.
-            return self.__getattribute__(attr_name)  # type: ignore
+        if attr_name in self.contracts:
+            source_id = self.contracts[attr_name].source_id
+            if source_id and not source_id.startswith(".cache"):
+                return ContractContainer(  # type: ignore
+                    contract_type=self.contracts[attr_name],
+                )
```

Comment:
probably should be an `else` and then raise `AttributeError` with the specific issue here (cannot load dependency contracts this way)